webapp/views/wpt-results.js

```diff
@@ -401,11 +401,11 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
       <wpt-metadata products="[[displayedProducts]]"
                     path="[[path]]"
                     search-results="[[searchResults]]"
-                    metadata-map="{{metadataMap}}"
-                    label-map="{{labelMap}}"
+                    metadata-map="[[metadataMap]]"
+                    label-map="[[labelMap}]]"
```

Comment:
wpt-results still need to add EventListener for `metadataMap` and `labelMap` because the data is being fetched inside of <wpt-metadata>. Once it is fetched, <wpt-metadata> notifies wpt-results that data has been fetched. Then wpt-results updates the inline labels UI in the test results.

Reply:
OK, done.